tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -71,3 +75,5 @@ void set_everything_c()
     set_invalid_c();
     set_inexact_c();
 }
+
+#endif // CPPUTEST_HAVE_FENV
```

Comment:
It seems Wcl objects to the missing newline at the end of file, not sure though.